microsoft-edge/dualengine/concepts/adapter-dll.md

```diff
@@ -19,7 +19,7 @@ To have Internet Explorer successfully load your DLL, do the following.
 <!-- ====================================================================== -->
 ## Unlock the Limited Access Feature
 
-The DualEngine API is a Limited Access Feature (LAF); that is, a feature that needs to be unlocked before it can be used. For more information about implementation, see [LimitedAccessFeatures Class](/uwp/api/windows.applicationmodel.limitedaccessfeatures). To request an unlock token, contact [Microsoft Support](https://aka.ms/LAFAccessRequests).
+The DualEngine API is a Limited Access Feature (LAF); that is, a feature that needs to be unlocked before it can be used. For more information about implementation, see [LimitedAccessFeatures Class](/uwp/api/windows.applicationmodel.limitedaccessfeatures). To request an unlock token, contact [Microsoft Support](https://go.microsoft.com/fwlink/?linkid=2271232&clcid=0x409).

```

Comment:
```suggestion
The DualEngine API is a Limited Access Feature (LAF); that is, a feature that needs to be unlocked before it can be used. For more information about implementation, see [LimitedAccessFeatures Class](/uwp/api/windows.applicationmodel.limitedaccessfeatures). To request an unlock token, fill in the form [Limited Access Feature APIs Request Access Token](https://go.microsoft.com/fwlink/?linkid=2271232&clcid=0x409).
```
The link does not end up at "Microsoft Support" in any discernable sense; the word "support" doesn't appear anywhere in the web page/browser.  
The resulting url is not "support", but office.com.  
Per style guides, link text should match the destination title, which is, per the tab title (= when you paste url into Word), "Limited Access Feature APIs Request Access Token".   Else disorienting nav UX.
Result: 
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/d92205af-18c8-427e-a5ba-4a649127d9fd)